doc-index-updater/overlays/non-prod/keys.sh

```diff
@@ -75,3 +75,13 @@ kubectl create secret generic storage-creds \
     --from-literal key="$BLOB_KEY" |
     kubeseal \
         --format yaml >SealedSecret-storage-creds.yaml
+
+# HTTP Basic Auth credentials...
+kubectl create secret generic basic-auth-creds \
+    -n doc-index-updater \
+    -o json \
+    --dry-run \
+    --from-literal username="username" \
+    --from-literal password="password" |
```

Comment:
I wonder if the default credentials should be something other than the names of the variables here? Just a little worried that we might introduce a bug where we use the variable names instead of variable values.

Reply:
imo these should come from a keyvault but...

[](!https://i.imgflip.com/3tckvi.jpg)